docs/BasicCommunication/OnAppDeactivated/index.md

```diff
@@ -24,6 +24,9 @@ In the event a navigation application is in `FULL` or `LIMITED` and the HMI send
 
 In the event a navigation application is in `FULL` or `Limited` and the HMI sends `OnAppDeactivated` (reason="AUDIO"), SDL must set the navigation app to `LIMITED` with the AudioStreamingState set to `AUDIBLE`.
```

Comment:
I cannot find the `reason` parameter in `OnAppDeactivated` RPC in HMI_API
```xml 
   <function name="OnAppDeactivated" messagetype="notification">
      <description>Must be sent by HU system when the user switches to any functionality which is not other mobile application.</description>
      <param name="appID" type="Integer" mandatory="true">
        <description>ID of deactivated application.</description>
      </param>
    </function>
```

Reply:
do you mean `OnEventChanged` with `eventName`=`AUDIO_SOURCE`?

Reply:
@yang1070 removed deactivate reason as not actual 388beda